butterfly-utilities/src/main/java/com/paypal/butterfly/utilities/operations/pom/PomRemoveDependencyExclusion.java

```diff
@@ -0,0 +1,165 @@
+import com.paypal.butterfly.extensions.api.operations.ChangeOrRemoveElement;
+import org.apache.maven.model.Dependency;
+import org.apache.maven.model.Exclusion;
+import org.apache.maven.model.Model;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+/**
+ * Removes an exclusion from a dependency in a POM file.
+ * If the POM file does not have the specified dependency, the operation will return an error.
+ * That behavior can be changed though, see {@link ChangeOrRemoveElement} for further details.
+ * @author asingh21
+ */
+public class PomRemoveDependencyExclusion extends AbstractArtifactPomOperation<PomRemoveDependencyExclusion> implements ChangeOrRemoveElement<PomRemoveDependencyExclusion> {
+
+    private static final String DESCRIPTION = "Remove the exclusion %s:%s if present in dependency %s:%s in POM file %s";
+
+    private Exclusion exclusion;
+
+    private ChangeOrRemoveElement.IfNotPresent ifNotPresent = ChangeOrRemoveElement.IfNotPresent.Fail;
+
+    public PomRemoveDependencyExclusion() {
+    }
+
+    /**
+     * Operation to remove an exclusion from a dependency in a POM file.
+     *
+     * @param groupId group id of the dependency to be replaced
+     * @param artifactId artifact id of the dependency to be replaced
+     * @param exclusionGroupId group id of the exclusion to remove
+     * @param exclusionArtifactId artifact id of the exclusion to remove
+     */
+    public PomRemoveDependencyExclusion(String groupId, String artifactId, String exclusionGroupId, String exclusionArtifactId) {
+        setGroupId(groupId);
+        setArtifactId(artifactId);
+        exclusion = new Exclusion();
+        setExclusionGroupId(exclusionGroupId);
+        setExclusionArtifactId(exclusionArtifactId);
+    }
+
+    public PomRemoveDependencyExclusion setExclusionGroupId(String exclusionGroupId) {
+        TransformationUtility.checkForBlankString("exclusionGroupId", exclusionGroupId);
+        this.exclusion.setGroupId(exclusionGroupId);
+        return this;
+    }
+
+    public PomRemoveDependencyExclusion setExclusionArtifactId(String exclusionArtifactId) {
+        TransformationUtility.checkForBlankString("exclusionArtifactId", exclusionArtifactId);
+        this.exclusion.setArtifactId(exclusionArtifactId);
+        return this;
+    }
+
+
+
+    @Override
+    public PomRemoveDependencyExclusion failIfNotPresent() {
+        ifNotPresent = ChangeOrRemoveElement.IfNotPresent.Fail;
+        return this;
+    }
+
+    @Override
+    public PomRemoveDependencyExclusion warnIfNotPresent() {
+        ifNotPresent = ChangeOrRemoveElement.IfNotPresent.Warn;
+        return this;
+    }
+
+    @Override
+    public PomRemoveDependencyExclusion noOpIfNotPresent() {
+        ifNotPresent = ChangeOrRemoveElement.IfNotPresent.NoOp;
+        return this;
+    }
+
+    public Exclusion getExclusion() {
+        return this.exclusion;
+    }
```

Comment:
Please remove this getter. I don't think it is necessary, as `exclusion` is used internally only.